doc/expressions.md

```diff
@@ -419,33 +419,28 @@ Every computable definition in Lean is compiled to bytecode at definition time.
 
 .. code-block:: lean
 
-    #reduce (λ x, x + 3) 5
-    #eval   (λ x, x + 3) 5
+    #reduce (fun x => x + 3) 5
+    #eval   (fun x => x + 3) 5
 
-    #reduce let x := 5 in x + 3
-    #eval   let x := 5 in x + 3
+    #reduce let x := 5; x + 3
+    #eval   let x := 5; x + 3
 
     def f x := x + 3
 
     #reduce f 5
     #eval   f 5
 
-    #reduce @nat.rec (λ n, Nat) (0 : Nat)
-                     (λ n recval : Nat, recval + n + 1) (5 : Nat)
```

Comment:
What is the motivation for removing this? The `#eval` won't work, but `#reduce` should.

Reply:
It's been reintroduced.